### PR TITLE
Artist name component

### PR DESCRIPTION
**Pull request by emeliese**
### Add ArtistName component

- Display ArtistsName with links to spotify via Album component
- Separate css file for ArtistName

### CODE REVIEW

File: src/components/Album.jsx [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
   // console.log(album);
 
-  const image = album.images.find(image => image.height === 640);
+  const image = album.images.find((image) => image.height === 640);
 
   return (
     <div className="album-container">
       <CoverImage img={image.url} alt={album.name} />
       {<AlbumName albumName={album.name} uri={album.uri} />}
+      {<ArtistName className="artist-name" artists={album.artists} />}
```
Comment (themisterkai) [inline src/components/Album.jsx]: we don't need to wrap this component in curly braces. We can also remove them for line 15.

File: src/components/ArtistName.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { PropTypes } from "prop-types";
+import "./ArtistName.css";
+
+export const ArtistName = ({ artists }) => {
+  return (
+    <div className="artist-container">
+      {artists.map((artist) => (
+        <span key={artist.id} className="artist-name-link">
+          <a href={artist.external_urls.spotify}>{artist.name} </a>
```
Comment (themisterkai) [inline src/components/ArtistName.jsx]: there's an extra space between `{artist.name}` and `</a>` on line 9. It's causing the artist name to be rendered somewhat incorrectly if there are 2 or more artists. Can we remove that space?


Screenshot 2023-10-21 at 20 47 27

File: src/components/ArtistName.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+.artist-name-link:not(:first-of-type)::before {
+  content: " , ";
```
Comment (themisterkai) [inline src/components/ArtistName.css]: This should be `", ";` (without a space between the opening `"` and the `,`)

File: src/components/ArtistName.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  color: #a0a0a0;
+}
+
+.artist-name-link:last-child::before {
+  content: " & ";
```
Comment (themisterkai) [inline src/components/ArtistName.css:7]: Awesome solution for this! 🙌🏽

File: src/components/ArtistName.jsx (hunk 2 above)
Comment (emeliese) [reply src/components/ArtistName.jsx]: Of course, I'll remove it

File: src/components/Album.jsx (hunk 1 above)
Comment (emeliese) [reply src/components/Album.jsx]: I see! For some reason I thought that was a React thing. Will remove them for bort components!

File: src/components/ArtistName.css (hunk 4 above)
Comment (emeliese) [reply src/components/ArtistName.css:7]: Thank you! Found a lot of good inpout on Technigo stack overflow

File: src/components/ArtistName.css (hunk 3 above)
Comment (emeliese) [reply src/components/ArtistName.css]: Fixed!

**Review by themisterkai — APPROVED** [review]

**Pull request merged**